src/agents/supervisor/prompts.py

```diff
@@ -1,112 +1,52 @@
-   - Do not add extra steps or elaborate beyond the original questions.
-   - Maintain the original wording and order of the questions.
-   - Mention the resource information like namespace, resource kind, api version and name if provided.
+Sample Queries and Responses:
 
-2. For unrelated queries:
-   - Provide a direct response without a plan/subtasks.
-   - Set the response attribute accordingly.
+- Kyma or Kubernetes related queries:
 
-3. Consider past conversations in your response.
+  Query: "What is Kyma serverless? what is the status of my cluster?"
+ 
+    "response": None,
+      "subtasks": [
+          ("description": "What is Kyma serverless?","assigned_to": "KymaAgent") ,
+          ("description": "what is the status of my cluster?","assigned_to": "KubernetesAgent")]
+          
+     
+  Query: "Create a hello world app and deploy it with Kyma?"
+  
+  "response": None,
+  "subtasks": [
+           ( "description": "Create a hello world app", "assigned_to": "Common"),
+           ("description": "deploy the app with Kyma","assigned_to": "KymaAgent")
+    ]
+ 
 
-4. Strictly adhere to the following response format:
-   {output_format}
+Guidelines:
 
-5. Assign each point/question to one of these agents: {members}.
+1. For queries about Kyma or Kubernetes create subtasks:
+   - Create a plan that directly mirrors the original query points.
+   - Mention the resource information like namespace, resource kind, api version and name if provided.
+2. Keep subtasks in same order as original query.
+3. Consider past conversations in your response.
 
 Key Principles:
 - Understand the query thoroughly.
 - Identify distinct questions or tasks within the query.
 - Preserve the original wording for each item.
-- Maintain the order of questions as presented in the query.
 - Keep the plan concise, avoiding any additional or explanatory steps.
 - Focus solely on the key points raised in the query.
 
-Sample Queries and Responses:
-- General queries that are irrelevant to Kyma or Kubernetes:
-  Query: "What is the capital of France?"
-  Output:
-  ```
-  {{
-    "response": "Paris"
-  }}
-  ```
-
-- Kyma or Kubernetes related queries:
-  Query: "What is Kyma function?"
-  Output:
-  ```
-  {{
-      "subtasks": [
-          {{
-              "description": "What is Kyma function?",
-              "assigned_to": "KymaAgent"
-          }}
-      ]
-  }}
-  ```
-
-  Query: "What is Kyma serverless? Why my k8s deployment is failing?"
-  Output:
-  ```
-  {{
-      "subtasks": [
-          {{
-              "description": "What is Kyma serverless?",
-              "assigned_to": "KymaAgent"
-          }},
-          {{
-              "description": "Why my k8s deployment is failing?",
-              "assigned_to": "KubernetesAgent"
-          }}
-      ]
-  }}
-  ```
+Agent Classification:
+- "{kymaAgent}": Manages Kyma specific topics
+- "{kubernetesAgent}": Handles Kubernetes related queries
+- "{commonAgent}": Covers all other general queries
```

Comment:
let's use variables instead of hardcode agent names?

Reply:
Yes these are variables only , i will change the casing